test_audit.py

```
@@ -0,0 +1,14 @@
+from resync.core.audit_db import _validate_audit_record
+
+# Test cases
+cases = [
+    ({"user_query": "test", "action": "TEST"}, "Memory ID is required"),
+    ({"id": "test", "action": "TEST"}, "User query is required"),
+]
+
+for record, expected in cases:
```

Ruff (0.15.0)

[warning] Loop control variable `expected` not used within loop body

Rename unused `expected` to `_expected`

(B007)